pegasus/sites.v3/code.org/public/teach.haml

```diff
@@ -21,6 +21,9 @@ theme: responsive_full_width
 
 = view :curriculum_launch_skinny_banner
 
+-# Show Hour of Code 2024 banner if hoc_mode is set to "soon-hoc"
+= view :hoc_skinny_banner
```

Comment:
Similar question here, would it be possible for us to end up with two back-to-back skinny banners?

Reply:
Same answer above, technically possible but unlikely to happen 🙌